SECURITY.md

```diff
@@ -1,5 +1,174 @@
-# Security Policy
+# Intro
 
-Sui is pre-release software under rapid development. It has not been audited and is not yet ready for production use.
+We appreciate your participation in keeping the Sui network secure. Please read all details on how to report an issue, what bug bounties are available, and who is eligible for bounties.
 
-After we have completed internal and external audits of Sui, we will establish a responsible disclosure policy and a bug bounty program covering both [protocol security](doc/src/learn/sui-security.md) and software security. In the meantime, please report security problems to security@mystenlabs.com.
+The program is run in partnership with [Immunefi](https://immunefi.com/). More information is available on [their site](https://immunefi.com/bounty/sui/).
+
+If you have any questions or concerns, please contact support@immunefi.com.
+
+# Reporting and response process
+
+All reports must be sent using Immunefi’s [secure dashboard](https://bugs.immunefi.com/). **Please DO NOT report a security issue using GitHub, email, or Discord.**
```

Comment:
I think we need something about how to report issues outside the scope of the bug bounty program. These used to go to security@mystenlabs.com, but it would be good to have a Sui Foundation alias for this.

Reply:
@wriches created a `security@sui.io`

Reply:
added it here: https://github.com/MystenLabs/sui/pull/10800/files#diff-f6ed156e4bf5c791680662464b94ea5d753f219ee816b385f67870e2c0d7d4c7R17